app/code/Magento/ImportExport/Block/Adminhtml/Export/Filter.php

```diff
@@ -419,13 +438,34 @@ public function getRowUrl($row)
     }
 
     /**
-     * Prepare collection by setting page number, sorting etc..
+     * Get Disabled Attributes
+     *
+     * @return array
+     */
+    private function getDisabledAttributes()
+    {
+        $disabledAttrs = [];
+        $productTypes = $this->exportConfig->getEntityTypes(CatalogProduct::ENTITY);
```

Comment:
The implementation in `ImportExport` module still aware of Catalog Product. I think it's better to move it to `Magento_CatalogImportExport` module.

Reply:
hi @sivaschenko 

Can you please suggest where i suppose to move these changes will be best.
**Magento\CatalogImportExport\Model\Export\Product**
initTypeModels() function does the same , can i make a public function to return `$this->_disabledAttrs`
I was thinking to move it in this model but i need your help.

Thank you.

Reply:
@sivaschenko 
any suggestions on this PR .
Thank you.